package.json

```diff
@@ -10,6 +10,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "axios": "^1.7.9",
     "dotenv": "^16.4.7",
     "ejs": "^3.1.10",
```

Comment:
why ejs? this is a really ancient package and an outdated way of creating dynamic content. I'd rather you use React, Vue, etc.